enroll/profile.go

```diff
@@ -77,3 +77,18 @@ type MDMPayloadContent struct {
 	ServerURL               string
 	Topic                   string
 }
+
+var DefaultProfileServiceDeviceAttributes = []string{
```

Comment:
This list is only being accessed once, so let's move it inside the `OTAEnroll` method. We can revisit this in the future as needed.